CRM/Eck/BAO/EckEntityType.php

```diff
@@ -19,25 +19,22 @@
 
 class CRM_Eck_BAO_EckEntityType extends CRM_Eck_DAO_EckEntityType {
 
-  protected static $_entityTypes;
```

Comment:
Using a static class variable is a bad idea because it interferes with unit testing and other long-running processes. There is no way to clear this cached value in case it gets stale (e.g. when adding/removing an entity type, this variable will not be updated!).
`Civi::$statics` is the solution. It automatically gets cleared between each unit test, and during a cache clear.